### PR TITLE
refactor: error message

### PR DESCRIPTION
**Pull request by alexander-akait**
**What kind of change does this PR introduce?**

refactor, it is regression from big refactor, problems doesn't have helpful information about possible values to fix, it should be implemented on webpack side (I requested it)

**Did you add tests for your changes?**

Yes

**If relevant, did you update the documentation?**

No need

**Summary**

No

**Does this PR introduce a breaking change?**

No

**Other information**

No

### CODE REVIEW

File: packages/webpack-cli/lib/webpack-cli.js [hunk 1]
```diff
@@ -256,8 +256,6 @@ class WebpackCLI {
                             }`,
                         );
                     });
-
-                    process.exit(2);
```
Comment (alexander-akait) [inline packages/webpack-cli/lib/webpack-cli.js:260]: We will get the exit code 2 from webpack (when we will validate options)

**Review by anshumanv — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Review by rishabh3112 — APPROVED** [review]
Need to fix `--parallelism` test, rest looks good.

**Comment by alexander-akait** [comment]
We need improve problems on webpack side, i.e. webpack should return helpful message

**Comment by alexander-akait** [comment]
Fixed in master, we have `expected` in problems

**Pull request closed without merging**